vivisect/symboliks/emulator.py

```diff
@@ -68,7 +67,7 @@ def applyEffects(self, effects):
         Apply the given effects to the emulator.  Return a list of updated
         effects which reflect the state during emulation.
         '''
-        return [ e.applyEffect(self) for e in effects ]
+        return [e.applyEffect(self) for e in effects]
```

Comment:
Yea. This probably falls under a larger "what is the coding standard for vivisect" question that needs to be addressed. 

I like flake8 just because I have the hope that one day it's more than you, me, and visi committing to vivisect, and flake8 is the officially "blessed" standard python supports, so it's easy to get people around that (and there's default tooling for it, etc). 

But honestly I don't have strong feelings beyond that. I've already swapped over a bunch of lines to this standard, so I'll leave it as is for this PR. But going forward we should have the discussion of what the vivisect standard should look like, and getting a flake8 config file for that set up.

Reply:
i understand all of that.  we're on the same page here.  i'm ok with it changing.  perhaps my eyes will learn to deal with that better anyway.